react-app/src/components/PatientData.js

```diff
@@ -1,15 +1,22 @@
 import React from 'react';
 import ReactJson from 'react-json-view';
+import LogList from './LogList';
 
 function PatientData(props) {
   return (
     <div>
       {props.id === null && (
         <div className="page-text">
-          <p>Select a patient to view their information.</p>
+          <p>Select a patient to view their information or click on "Log File" to view the logger messages.</p>
         </div>
       )}
-      {props.id !== null && (
+      {props.id !== null && props.id === -1 && (
+        <div>
+          <h3 className="page-subtitle">Log File</h3>
+          <LogList loggedMessages={props.loggedMessages} />
```

Comment:
This is pretty minor, but is it possible to have the list of log messages scroll while keeping the "Log File" title static at the top of the screen? I think that might be nice to have since the list is usually so long and the "Log File" feels like a header that usually stays static at the top of the page. If that's really difficult CSS-wise, we can evaluate if it's actually necessary.

Reply:
I tried making the title sticky, but I think the grid system interferes with the display somehow. The ListGroup covers it up so that the sticky title only appears while scrolling.

Reply:
Hm I feel like there should be a way to make the header stay at the top of the screen. I think having it stay at the top will make it feel a little nicer for users too. If the grid system doesn't let the top "Log File" header stay sticky, there is probably a way to set the height of the overall page so that the page stays at a 100% height and doesn't scroll, and then you can set the div that contains the Log messages (or the top level div within the LogList component) so that that div takes up the remaining space on the screen and any overflow within that should scroll.

If you want any help trying out those approaches, let me know.

Reply:
I tried to make the title sticky in the ExtractionError component of the error-page branch, which doesn't use the grid system, and it looks like the issue is actually that the ListGroup covers the sticky component. Additionally, I realized while looking into this that the sticky title is basically a NavBar with a white background. Maybe I could use a NavBar instead? Since it's a react-bootstrap component, maybe the ListGroup issue will go away.

Reply:
Making a NavBar with a white background is definitely another approach! I think typically NavBars are used for a full app, rather than just a title on a specific page, so I'm not completely sure if that's the approach we want to go with in the end, but it's definitely worth exploring and seeing generally other ways to position the elements.

The thought process I was having earlier was to make sure that each page of the whole app only takes up 100% of the height of the app. Then any components on that page that need more room, like the LogList, can scroll within that space, so they don't extend the page any longer. Just another possible approach to explore!

Reply:
Alright, that makes sense. I'll try implementing that.

Reply:
Solved this by moving the subtitle after the ListGroup so that the ListGroup scrolls behind it instead of in front of it.

Reply:
Oh interesting. I pulled the branch and this definitely works how I was expecting it to in the app. However, I think having the `<h3>` below the `<LogList>` is confusing to read in the code. Is there a way to make just the <LogList> component scrollable so we can maintain the order in the code that matches the rendered order in the app?

Reply:
Cool! This looks much nicer to me, and the code is much more readable! Thanks for spending the time to look into this!